cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/nukkit/NukkitCloudNetBridgePlugin.java

```diff
@@ -39,4 +42,42 @@ private void initListeners() {
         CloudNetDriver.getInstance().getEventManager().registerListener(new NukkitCloudNetListener());
         CloudNetDriver.getInstance().getEventManager().registerListener(new BridgeCustomChannelMessageListener());
     }
+
+    private void runFireServerListPingEvent() {
+        Server.getInstance().getScheduler().scheduleRepeatingTask(this, () -> {
```

Comment:
Please do this like it's done for Bukkit (the Scheduler is started in the onEnable-method) or change it in the Bukkit plugin.

Reply:
I just left it as it was in Bukkit, but right now in Bukkit it is just complete ...

Reply:
In bukkit, the ``runFireServerListPingEvent()``-method doesn't start the scheduler, it just calls the event one time. The scheduler ist started in the onEnable and executes the ``runFireServerListPingEvent()``-method.

Reply:
It's stupid to create a sync executor which then calls the event async...

Reply:
You don't understand me, please look at this class: https://github.com/CloudNetService/CloudNet-v3/blob/master/cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/bukkit/BukkitCloudNetBridgePlugin.java

You start the Scheduler IN the ``runFireServerListPingEvent()``-method while bukkit just calls the event ONCE when executing this method.
The structure and logic of the Nukkit and Bukkit plugins should be the same.

Reply:
I don't think you understand my sentence. A SYNC repeating task is started which then calls the method which then calls the event ASYNC called which is completely useless? Why not just do an ASYNC repeating task?

Reply:
Yeah, this is pretty stupid, but I've never said that you should do it like this too.
You might just change to bukkit scheduler to a async scheduler and then remove the async task execution.